index.js

```diff
@@ -1,17 +1,51 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(pos > this.items.length) {
+      throw new Error("OutOfBounds");
+    } else {
+      return this.items[pos];
+    }
+    
+  }
 
-  max() {}
+  max() {
+    if(this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.max(...this.items);
```

Comment:
```suggestion
      return Math.max(...this.items);
```

Esto te lo doy por bueno si sabes decirme que son los tres puntos y porque son necesarios en esta operacion :D